test/e2e/gql/fixture/src/query.gql

```diff
@@ -1,4 +1,4 @@
-# import './fragment.gql'
```

Comment:
Was this working before?

Reply:
No - the test was broken also. I fixed the test and make it more robust to catch this kind of failure